ansible/configs/ansible-multi-node/sample_vars/aapf-osp-vars.yml

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
@tonykay, you may wish to change file name from aapf-osp-vars.yml to osp-vars.yml btw this is not mandatory.